orders/views.py

```diff
@@ -1,3 +1,59 @@
-from django.shortcuts import render
+import uuid
 
-# Create your views here.
+from datetime         import datetime
+from django.views     import View
+from django.http      import JsonResponse
+from django.db        import transaction
+from django.db.models import Sum,F
+
+from core.utils       import login_confirm
+from orders.models    import Order, OrderStatus, OrderItem
+from carts.models     import Cart
+from users.models     import User
+
+class OrderView(View):
+
+    @login_confirm
+    def post(self, request, *args, **kwargs):
+        user     = request.user
+        carts = Cart.objects.filter(user_id = request.user.id)
+        if not carts.exists():
+            return JsonResponse({"message" : "NOT EXIST CARTS"}, status=400)
```

Comment:
1. 컨벤션 맞춰주세요.
2. 장바구니 전체에 대해서 존재 유무를 판단하는데, 구매하려고 하는 장바구니 정보의 유무를 판단해 주세요.
```suggestion
        user  = request.user
        carts = Cart.objects.filter(user = request.user)
       
         if not carts.exists():
            return JsonResponse({"message" : "NOT EXIST CARTS"}, status=400)
```